docs/meshstack.emergency-users.md

```diff
@@ -0,0 +1,98 @@
+
+Since emergency access should only be temporary, it's strongly advised to assign the user a role with a [set expiration date](./meshcloud.project.md#expiry-of-a-user-assignment) which will ensure that the user is automatically removed from the project after the specified date.
+
+### Approving emergency user requests
+
+User project role assignments can be configured to require consent from multiple customer admins ([4 eye principle](./meshstack.authorization.md#user-project-role-approval)). To avoid situations where not enough customer admins are available to confirm an urgent user role request, the meshPartner can confirm project role requests directly:
+
+- Ensure that the partner is selected from the customer drop down
+- Open "Administration" from the settings menu in the top right
+- Navigate to "Customers" and select "User Pending Role Requests" from the actions column for the customer to which the project is assigned
+- Approve the user role request
+
+### Removing emergency user via Customer
+
+When emergency access is no longer required the following steps will revert performed  changes:
+
+- Remove user from project via Account → Projects → Edit Project (performed automatically if expiration date was set)
+- Remove user from customer via Account → Users
+
+## Access with Partner
+
+Even when no customer admin is available, users with partner admin/employee access can manage permissions for their ,managed meshCustomers.
+
+### Adding emergency user as Partner
+
+In order to manage users for a specific project, the partner user must add their own account to the respective customer as a customer admin:
+
+- Ensure that the partner is selected
+- Open "Administration" from the settings menu in the top right
+- Navigate to "Customers" and select "Customer User" from the actions column for the target customer
+- Activate “Add Myself” button in the top right
+  
+Afterwards they may proceed to manage users for this customer as a customer admin (see previous section).
+
+### Removing emergency user as Partner
+
+Since the partner user is now a customer admin the procedure is the same as outlined in the previous section.
+However, to revert the temporary customer admin role assignment, another customer admin must remove the partner user from the customer via Account → Users.
+
+## Access Through Service Users
+
+When access is only required for cloud resources and the cloud platform supports [service users](./meshcloud.service-user.md) emergency access can also be arranged by utilizing service users.
+Service user credentials can be generated by any user assigned to a project by navigating to the desired project and platform and selecting “Service Users”. When they are no longer required they can be deleted here as well.
+
+## Auditing Emergency Access
+
+Since customer/project access permissions should not be granted lightly all changes to them are logged and can be audited by partner admin/employee users.
```

Comment:
```suggestion
Since meshCustomer/meshProject access permissions should not be granted lightly all changes to them are logged and can be audited by partner admin/employee users.
```